components/script/dom/mutationrecord.rs

```diff
@@ -0,0 +1,135 @@
+use core::ptr;
+use dom_struct::dom_struct;
+use dom::bindings::codegen::Bindings::MutationRecordBinding::MutationRecordBinding::MutationRecordMethods;
+use dom::bindings::js::{JS, Root};
+use dom::bindings::str::DOMString;
+use dom::bindings::reflector::Reflector;
+use dom::node::Node;
+use dom::nodelist::NodeList;
+use dom::window::Window;
+use std::default::Default;
+
+#[dom_struct]
+pub struct MutationRecord {
+  reflector_: Reflector,
+
+//  readonly attribute DOMString record_type;
```

Comment:
Rather than duplicating IDL constructs in the comments, we should be describing what these fields used are for in English prose instead.